gcloud/datastore/test_client.py

```diff
@@ -18,29 +18,44 @@
 class TestClient(unittest2.TestCase):
 
     DATASET_ID = 'DATASET'
+    CONNECTION = object()
```

Comment:
I'd like to do a sweep through and make this consistent throughout (at least for clients).